.circleci/config.yml

```diff
@@ -1,158 +1,165 @@
 version: 2.1
 orbs:
   aws-s3: circleci/aws-s3@2.0.0
-
-commands:
-  source-vars:
-    description: source the environment variables from the temp workspace
-    steps:
-      - run: |
-          set -x
-          cat /tmp/workspace/env_vars.sh >> $BASH_ENV
-          source $BASH_ENV
+parameters:
+  cross-container-tag:
+    type: string
+    default: go1.15.10-ec819fab057e2efdcf478fec05bb5533957721ec
```

Comment:
Can you add a comment that explains how this tag is derived, so that it can be updated later when necessary?

Presumably the idea is to pass this param dynamically when building OSS from Plutonium CI?

It might also be a good idea to add a comment in the go.mod where go 1.15 is specified, with instructions to update this tag at least whenever the go version is bumped. Otherwise this will get missed.

Reply:
Fixed by https://github.com/influxdata/influxdb/pull/22333

In Enterprise, we assert that we build with the same container as OSS but do not pass anything dynamically.